server/expression/data_store.go

```diff
@@ -99,5 +99,5 @@ func (ds EnvironmentDataStore) Get(name string) (string, error) {
 		}
 	}
 
-	return "", fmt.Errorf(`environment variable "%s" not found`, name)
+	return "", fmt.Errorf(`variable "%s" not found`, name)
```

Comment:
We should rename the `EnvironmentDataStore` to `VariableSetDataStore` or something similar

Reply:
Rest looks good

Reply:
Done! Do you mind retaking a look?